Makefile.doc

```diff
@@ -56,9 +56,9 @@ DOCCOMMON:=doc/common/version.tex doc/common/title.tex doc/common/macros.tex
 ######################################################################
 
 .PHONY: doc doc-html doc-pdf doc-ps
-.PHONY: stdlib full-stdlib sphinx
+.PHONY: doc-stdlib full-stdlib sphinx
```

Comment:
If you rename the `stdlib` target, you should rename `full-stdlib` as well. But is this really a good idea to rename these targets (may break some existing packaging scripts) given that this legacy Makefile is going away in favor of Dune soon?

Reply:
Thanks for the review @Zimmi48 , I kept this particular renaming as the target did interact badly with having a `stdlib` directory which I thought could happen at some point in the future.

I dunno indeed; any of the choices [fixing the doc or reverting the target renaming] is fine to me.

Reply:
OK I get it, thanks! Renaming makes sense in this case. I think I would prefer `stdlib-doc` (sounds closer to English) but not a big deal.

Reply:
Doc fixed, thanks; I've kept the `doc-` prefix as there are some other `doc-` targets already available.